scripts/tablets/topology.py

```
@@ -0,0 +1,1048 @@
+        return False
+
+
+class LiveClusterTopologySource(TopologySource):
+    """
+    Topology is constructed by querying system tables from a live cluster.
+    """
+
+    def __init__(self, cluster, session):
+        self.cluster = cluster
+        self.session = session
+        self.queries = {table: session.prepare(f"SELECT * FROM {table.name}")
```

Ruff (0.16.3)

[error] Possible SQL injection vector through string-based query construction

(S608)